src/libPersistence/BlockStorage.cpp

```diff
@@ -186,6 +191,11 @@ bool BlockStorage::GetTxBlock(const boost::multiprecision::uint256_t& blockNum,
 bool BlockStorage::GetTxBody(const dev::h256& key, TxBodySharedPtr& body)
 {
 #ifndef IS_LOOKUP_NODE
+    if (!m_txBodyDBs.size())
```

Comment:
use m_txBodyDBs.empty() or !m_txBodyDBs.size() == 0

Use ! for boolean  and == for numerical comparsion